Src/Notion.Client/Models/PropertyValue/DatePropertyValue.cs

```diff
@@ -13,8 +13,8 @@ public class DatePropertyValue : PropertyValue
         /// <summary>
         /// Date
         /// </summary>
-        [JsonProperty("date")]
-        public Date Date { get; set; }
+        [JsonProperty("date", NullValueHandling = NullValueHandling.Include)]
+        public Date? Date { get; set; }
```

Comment:
I want to understand what is the reason behind adding nullable operator on a class?

Reply:
Hi @KoditkarVedant ,

As per the issue:  #237 , Notion API allows us to clear the Date Property completely. Instead of individually making Start Date and EndDate as null.

![image](https://user-images.githubusercontent.com/27886758/179337599-ce5d3fbb-1d4f-4375-a322-7f9fdb84d2b3.png)

As per my understanding, Date class is an exact map to the API Resource Value of Date , so making it as nullable was the only way to allow clearing the date. We cannot make DatePropertyValue  nullable as we still need the property id and type. 

![image](https://user-images.githubusercontent.com/27886758/179337684-9d1aaafe-28fa-417c-8abb-3c1efcd60afa.png)

Reply:
@vijayvarmaD I understand but `Date` is a class so it can be set to `null`. I am not sure why we need to user C# 8 and then add `?` to class?

Reply:
Ok